In [3]:
import math
""" Confidence Interval of 95%  Function

mean = the population mean
std = standard deviation of the population
sample_size = size of the sample

"""
#Standard error method that comes into computation
def standard_error(std, sample_size):
    return (std/(math.sqrt(sample_size)))


def ci_95(mean, se):
    z = 1.96

    ci_minus = mean - (z * se)
    ci_plus = mean + (z * se)
    ci = (ci_minus,ci_plus)
    #returns a tuple
    return ci

# 4.7 Chronic illness

**Given Data :** 
<br/>
point estimate = 45%<br/>
Standard Error = 1.2%

Creating a 95 % confidence interval for the proportion of US adults with one pr more chronic conditions.

In [4]:
ci_95(45,1.2)

(42.648, 47.352)

**<u>Inference : </u> : ** We are 95% confident that the proporion of U.S. citizen living with one or more chronic conditions is between 42.648 and 47.352 